src/latch_cli/services/launch/launch_v2.py

```diff
@@ -19,13 +23,126 @@
 from latch_sdk_config.latch import config
 from latch_sdk_gql.execute import execute
 
+ExecutionStatus = typing.Literal[
+    "UNDEFINED",
+    "QUEUED",
+    "RUNNING",
+    "SUCCEEDED",
+    "ABORTED",
+    "FAILED",
+    "INITIALIZING",
+    "WAITING_FOR_RESOURCES",
+    "SKIPPED",
+    "ABORTING",
+]
+
+
+class Execution:
+    def __init__(self, execution_id: str, python_outputs: dict[str, type]) -> None:
+        self.execution_id = execution_id
```

Comment:
```suggestion
        self.id = execution_id
```
so u don't the `@property` fn below